webrtc-c/canary/src/Config.cpp

```diff
@@ -0,0 +1,85 @@
+#include "Include.h"
+
+namespace Canary {
+
+STATUS mustenv(CHAR const* pKey, const CHAR** ppValue)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+
+    CHK(ppValue != NULL, STATUS_NULL_ARG);
+
+    CHK_ERR((*ppValue = getenv(pKey)) != NULL, STATUS_INVALID_OPERATION, "%s must be set", pKey);
+
+CleanUp:
+
+    return retStatus;
+}
+
+BOOL getenvBool(CHAR const* pKey)
+{
+    PCHAR pValue;
+
+    pValue = getenv(pKey);
+    if (pValue == NULL) {
```

Comment:
Why not use mustenv here

Reply:
So that we don't need to set all the values. But, you're right, this is for canaries, so it's much better to be more strict about this.